From ad069fadfcce2cf70f45b1c4a42665448675297e Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Mon, 12 Dec 2022 13:10:19 +0100
Subject: [PATCH] src/shared/util.c: include linux/limits.h

MAX_INPUT is defined in that file. This matters on non-glibc
systems such as those using musl.

Upstream-Status: Submitted [to linux-bluetooth@vger.kernel.org,luiz.von.dentz@intel.com,frederic.danis@collabora.com]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>

---
 src/shared/util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/util.c b/src/shared/util.c
index 34491f4..412f3ad 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -23,6 +23,7 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <limits.h>
+#include <linux/limits.h>
 #include <string.h>
 
 #ifdef HAVE_SYS_RANDOM_H
